modules/virt-applying-node-placement-rules.adoc

```diff
@@ -6,7 +6,12 @@
 [id="virt-applying-node-place-rules_{context}"]
 = Applying node placement rules
 
+ifndef::openshift-rosa,openshift-dedicated
```

Comment:
why you can't configure node placement in rosa?

Reply:
@orenc1 I should have added a note. I hid only editing a subscription to apply node placement rules. Does that make more sense?